client/ayon_core/plugins/publish/integrate_review.py

```diff
@@ -0,0 +1,102 @@
+import os
+
+import pyblish.api
+import ayon_api
+from ayon_api.server_api import RequestTypes
+
+from ayon_core.lib import get_media_mime_type
+from ayon_core.pipeline.publish import get_publish_repre_path
+
+
+class IntegrateAYONReview(pyblish.api.InstancePlugin):
+    label = "Integrate AYON Review"
+    # Must happen after IntegrateNew
```

Comment:
I'm pretty sure IntegrateNew does not exist anymore. :)
```suggestion
    # Must happen after IntegrateAsset
```